boot/build.go

```diff
@@ -131,8 +131,12 @@ func (b Build) Build(context libcnb.BuildContext) (libcnb.BuildResult, error) {
 		result.Layers = append(result.Layers, classpathLayer)
 
 	} else {
+		scbJarFound := FindExistingDependency(d, "spring-cloud-bindings")
+		if scbJarFound == true {
```

Comment:
```suggestion
		if scbJarFound {
```
Small simplification!